src/pages/index.js

```diff
@@ -15,28 +15,26 @@ const featureRows = [
       imageUrlDark: 'img/community_dark.svg',
       link: 'docs/chain/collator',
       description: (
-        <>
-          Become a collator in for our Peregrine Testnet and later help
-          decentralizing the KILT Blockchain.
-        </>
+        <>Become a collator and help decentralizing the KILT Blockchain.</>
       ),
     },
     {
       title: 'Workshop',
       imageUrl: 'img/expert_light.svg',
       imageUrlDark: 'img/expert_dark.svg',
       link: 'docs/sdk/workshop/welcome',
-      description: (
-        <>Explore the most fundamental features of the KILTprotocol.</>
-      ),
+      description: <>Claim, Attest and verify in our SDK workshop.</>,
```

Comment:
We either capitalize all these verbs or none (except for the first word which starts the sentence in this case)

```suggestion
      description: <>Claim, attest and verify in our SDK workshop.</>,
```